autoPyTorch/pipeline/components/setup/lr_scheduler/CosineAnnealingWarmRestarts.py

```diff
@@ -16,7 +16,7 @@
 class CosineAnnealingWarmRestarts(BaseLRComponent):
     """
     Set the learning rate of each parameter group using a cosine annealing schedule,
-    where \eta_{max}ηmax is set to the initial lr, T_{cur} is the number of epochs
```

Comment:
Why did you delete `\` ?

The usage of the Inline math represetations is nicely described [here](https://numpydoc.readthedocs.io/en/latest/format.html#notes).

```
where :math:`\eta_{\max}` is set to the initial lr, :math:`T_{\text{cur}}` is the number of epochs
since the last restart and :math:`T_{i}` is the number of epochs between two warm
```

Reply:
Hello, sorry this was not an intentional change. 

My vim configuration changed this. It did so because this should be a raw string, as `\e` is not a valid escaped sequence.